pkg/controllers/netpol/policy.go

```diff
@@ -85,58 +85,60 @@ func (npc *NetworkPolicyController) syncNetworkPolicyChains(networkPoliciesInfo
 		klog.V(1).Infof("Returned ipset mutex lock")
 	}()
 
-	ipset, err := utils.NewIPSet(false)
-	if err != nil {
-		return nil, nil, err
-	}
-	err = ipset.Save()
-	if err != nil {
-		return nil, nil, err
-	}
-	npc.ipSetHandler = ipset
-
 	activePolicyChains := make(map[string]bool)
 	activePolicyIPSets := make(map[string]bool)
 
-	// run through all network policies
-	for _, policy := range networkPoliciesInfo {
+	for ipFamily := range npc.ipFamilyHandlers {
+		ipset, err := utils.NewIPSet(ipFamily)
```

Comment:
pkg/agent/netpol/policy.go:96:32: cannot use ipFamily (variable of type *ipFamilyHandler) as uint16 value in argument to utils.NewIPSet (typecheck)
		ipset, err := utils.NewIPSet(ipFamily)